scheduler/src/cook/tools.clj

```diff
@@ -71,6 +71,11 @@
           db)
        (map first)))
 
+(defn get-quota-resource-types
+  "Return a list of resource types used for quota, :cpus :mem :gpus ..."
+  [db]
+  (remove #{:disk} (get-all-resource-types db)))
```

Comment:
Can this be deleted since it's defined in `queries`?

Reply:
yes, removed